toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarRenderer.kt

```diff
@@ -132,10 +132,11 @@ internal fun GraduatedLineScalebar(
     maxWidth: Float,
     tickMarks: List<ScalebarDivision>,
     colorScheme: ScalebarColors,
+    labelTypography: LabelTypography
```

Comment:
it seems that we only need the property `LabelTypography.labelStyle`, 
I think we should only pass necessary values. 
```suggestion
    labelStyle: TextStyle
```